src/main/java/core/basesyntax/io/FileWriterImpl.java

```
@@ -0,0 +1,16 @@
+package core.basesyntax.io;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String reportGenerator, File output) {
```

Add explicit null checks at the beginning of the write method. The task requires: "Handle invalid inputs robustly by throwing RuntimeException with informative messages rather than printing or swallowing exceptions: ... Null parameters". Validate both the content (currently named `reportGenerator`) and the output path/file and throw informative RuntimeException before attempting to open streams.